<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>facultyView</class>
 <widget class="QWidget" name="facultyView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>581</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome Faculty</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>591</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>581</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>40</y>
        <width>511</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="manageCoursesButton">
         <property name="text">
          <string>Manage Courses</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewCoursesButton">
         <property name="text">
          <string>View Courses</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="changePassButton">
         <property name="text">
          <string>Change Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logoutButton_2">
         <property name="text">
          <string>Logout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableView" name="scheduleTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>581</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="scheduleLabel">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>49</y>
       <width>571</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="midLineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>Current Student Semester Schedule</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>591</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
